core/src/main/java/com/datastrato/gravitino/storage/kv/KvGarbageCollector.java

```diff
@@ -188,6 +188,7 @@ private void collectAndRemoveOldVersionData() throws IOException {
 
       // All keys in this transaction have been deleted, we can remove the commit mark.
       if (keysDeletedCount == keysInTheTransaction.size()) {
+        LOG.info("Physically delete commit mark: {}", Bytes.wrap(kv.getKey()));
```

Comment:
This is an unrelated change, please remove this.

Reply:
OK